src/AvtaleSide/steg/BeregningTilskudd/VisUtregningenPanel.less

```diff
@@ -0,0 +1,71 @@
+@import '~nav-frontend-core/less/_variabler';
+
+.visUtregningenPanel {
+    &__rad {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        flex-direction: row;
+        width: 100%;
+        border-bottom: 1px solid @navGra20;
+        padding: 0.9rem 0 0.9rem 0;
+
+        &__siste {
+            border-bottom: 1px solid;
+        }
+
+        &__oppsummering {
+            border-bottom: none;
+        }
+    }
+    &__column {
+        &__siste {
+            display: flex;
+            justify-content: flex-end;
+        }
+    }
+    &__tittel {
+        display: flex;
+        align-items: center;
+    }
+
+    &__ikon {
+        margin-right: 1rem;
+    }
+
+    &__orienter {
+        &__sum {
+            &__tittel {
+                padding-left: 3rem;
+            }
+        }
+    }
+
+    &__orienter__left {
+        margin-left: 6rem;
+    }
+
+    // spesielt til mobil
+    @media (max-width: 755px) {
```

Comment:
Hvor har du tatt `755px` fra? 🙂
Etter et lite søk ser jeg vi flest ganger har brukt `@media (max-width: @screen-sm-min)` for mobil.

Reply:
husker ikke akkurat, fant det et sted jeg. 
Men det er sikkert mye bedre å bruke sm-min :)